### ISSUE TITLE
about the validation set

### ISSUE THREAD

**Issue by Zoeeeing** [OpenDriveLab/DriveLM#12]
Thanks for your great work! May I ask when you plan to release the validation set? Or can you provide the tokens of the validation samples?

**Comment by ChonghaoSima** [comment]
We may not release the annotation on nuscenes validatioin set as we may keep it as the "test" set for our AD challenge on CVPR this year.

**Comment by Zoeeeing** [comment]
Thanks for your response! And Could you provide the tokens of these key frames?

**Comment by ChonghaoSima** [comment]
If you mean the frame tokens used as the same in the DriveLM-nuScenes train json (as well as in the original nuScenes), we don't have it yet as we're still polishing the val data json file. For now you could use [this pickle file](https://drive.google.com/file/d/1muUnhFk0TpdiFVWBekwIUSjXzj7kUpt5/view?usp=sharing) as it contains the cam_front timestamp of the key frames in val. We will release the well-prepared file once the challenge test server open.

**Comment by Zoeeeing** [comment]
Thanks so much.

**Comment by Zoeeeing** [comment]
Sorry to bother you again. Is the current released dataset missing the motion labels mentioned in the paper? Or are these labels same with the labels in the Uniad?

**Comment by ChonghaoSima** [comment]
They are the same with the labels used in UniAD.

Best
Chonghao Sima
________________________________
